.env.tpl

```
@@ -0,0 +1,46 @@
+SKIP_PREFLIGHT_CHECK=true
+LOCAL_LOGIN=true
+attachmentsBucketName=op://mdct_devs/carts_secrets/attachmentsBucketName
+fiscalYearTemplateBucketName=op://mdct_devs/carts_secrets/fiscalYearTemplateBucketName
+loggingBucket=op://mdct_devs/carts_secrets/loggingBucket
+acsTableName=local-acs
+fmapTableName=local-fmap
+uploadsTableName=local-uploads
+stateTableName=local-state
+stateStatusTableName=local-state-status
+sectionBaseTableName=local-section-base
+sectionTableName=local-section
+stageEnrollmentCountsTableName=local-stg-enrollment-counts
+DYNAMODB_URL=http://localhost:8000
+COGNITO_USER_POOL_ID=op://mdct_devs/carts_secrets/COGNITO_USER_POOL_ID
+COGNITO_USER_POOL_CLIENT_ID=op://mdct_devs/carts_secrets/COGNITO_USER_POOL_CLIENT_ID
+POST_SIGNOUT_REDIRECT=op://mdct_devs/carts_secrets/POST_SIGNOUT_REDIRECT
+API_URL=http://localhost:3030/local
+S3_LOCAL_ENDPOINT=http://localhost:4569
+S3_ATTACHMENTS_BUCKET_NAME=op://mdct_devs/carts_secrets/S3_ATTACHMENTS_BUCKET_NAME
+URL=http://localhost/3000
+docraptorApiKey=op://mdct_devs/carts_secrets/docraptorApiKey #pragma: allowlist secret
+iamPath=/
+iamPermissionsBoundary="bound"
+DYNAMO_TABLE_ARN=local_nonsense_if_unset_we_search_CF_for
+SKIP_PREFLIGHT_CHECK=true
+SLS_INTERACTIVE_SETUP_ENABLE=1
+acsTableArn=local_nonsense_if_unset_we_search_CF_for
+fmapTableArn=local_nonsense_if_unset_we_search_CF_for
+sectionBaseTableArn=local_nonsense_if_unset_we_search_CF_for
+stateStatusTableArn=local_nonsense_if_unset_we_search_CF_for
+stateTableArn=local_nonsense_if_unset_we_search_CF_for
+stageEnrollmentCountsTableArn=local_nonsense_if_unset_we_search_CF_for
+sectionTableArn=local_nonsense_if_unset_we_search_CF_for
+uploadsTableArn=local_nonsense_if_unset_we_search_CF_for
+princeApiHost=op://mdct_devs/carts_secrets/princeApiHost
+princeApiPath=/doc-conv/508html-to-508pdf
+warmupSchedule="60 minutes"
+warmupConcurrency=5
+runV2DataMigration=false
```

I don't see any references to these in the codebase
```suggestion
SLS_INTERACTIVE_SETUP_ENABLE=1
```